### PR TITLE
LIVE-13135: fix overflow details on swap device confirmation

### PR DESCRIPTION
**Pull request by adumitrana-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Fix an issue where long texts could overflow on the swap device confirmation drawer

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)
 

https://github.com/LedgerHQ/ledger-live/assets/167981611/7b41a86e-eba6-4987-b004-f4203c8fc158

Verify the Swap details on your device before

Verity the Swap details on your device before

### CODE REVIEW

**Review by jdabbech-ledger — DISMISSED** [review]

**Review by liviuciulinaru — DISMISSED** [review]

**Review by mcayuelas-ledger — DISMISSED** [review]
LGTM for Hub side

**Review by liviuciulinaru — DISMISSED** [review]

**Review by jdabbech-ledger — DISMISSED** [review]

**Review by jdabbech-ledger — DISMISSED** [review]

**Review by ypolishchuk-ledger — APPROVED** [review]

**Review by jdabbech-ledger — APPROVED** [review]

**Review by liviuciulinaru — APPROVED** [review]

**Review by ypolishchuk-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/tests/specs/services/swap.spec.ts-snapshots/initiate-swap-linux.png
[the remark this answers is not in the document]
Comment (adumitrana-ledger) [reply apps/ledger-live-desktop/tests/specs/services/swap.spec.ts-snapshots/initiate-swap-linux.png:1]: We've added scroll support for the drawer, I'll have a look at the spacing and fix it 🙏, is ok If we have a separate PR for the fix?

**Comment by adumitrana-ledger** [comment]
@adumitrana-ledger

[Generating screenshots: workflow started](https://github.com/LedgerHQ/ledger-live/actions/runs/9839835802)

**Review by ypolishchuk-ledger — APPROVED** [review]

**Review by liviuciulinaru — APPROVED** [review]

**Review by jdabbech-ledger — APPROVED** [review]

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**